utils/dashboard.js

```diff
@@ -112,7 +112,34 @@ const dashboard = async (req, res) => {
     } else if (api === "dryRunNotificationSchema") {
         const { dryRunNotificationSchema } = require('./notifications');
         return await dryRunNotificationSchema(req, res);
-    }
+    } else if (api === 'resetUser') {
+        if (req.method !== 'POST') {
+          return res.status(405).json(getResponseJSON('Only POST requests are accepted!', 405));
+        }
+        // Only permit for dev apps
+        if (process.env.GCLOUD_PROJECT === 'nih-nci-dceg-connect-dev') {
+          let body = req.body;
+          if (!body.uid) {
+              return res.status(405).json(getResponseJSON('Missing UID!', 405));
+          }
+          let uidToReset = body.uid;
+          let saveToDb = body.saveToDb === 'true';
```

Comment:
is there a scenario where this wouldn't be true?

Reply:
Mostly in the event that we want to display a "dry run" report to the end user where they would see the data the new user would have before confirming the change. This feature isn't in the MVP front-end but I figured there were enough odds someone would want it to future-proof it.